test/mongodb.test.js

```diff
@@ -3372,6 +3372,143 @@ describe('mongodb connector', function() {
     });
   });
 
+  describe('Test processMongoDBUrl function', function() {
```

Comment:
Feel free to ignore this:

A trick that we use in new tests in LB4  is to programmatically generate `it()` using a for-loop and a list(s) of valid value so that we can test every reasonable permutation. This can be considered out of scope for this PR as I think these tests should be sufficient for now.

Reply:
Good to know, thanks!